app/src/main/java/com/group80/uoftinder/create_account_use_case/CreateAccountPresenter.java

```diff
@@ -0,0 +1,21 @@
+package com.group80.uoftinder.create_account_use_case;
+
+public class CreateAccountPresenter {
```

Comment:
Considering all the code are commented out, and I don't think we are using this presenter anywhere, can we remove this file? (see also [this comment](https://github.com/CSC207-2022F-UofT/course-project-group-80/pull/24/files#r1027251277) in [CreateAccountView.java](https://github.com/CSC207-2022F-UofT/course-project-group-80/pull/24/files#diff-ece65604b90785502ed00db116e1ffe3e52381577a53ae7c8040bc2278709474))

Reply:
For now no. Originally I had methods in my presenter but Jessica/I made some major changes to my code which resulted in methods in my Presenter being commented out. However, I am planning on moving a lot of my functions from CreateAccountView to Presenter - for clean architecture.